### PR TITLE
Route based subviews

### PR DESCRIPTION
**Pull request by anttimaki**

### CODE REVIEW

**Comment by anttimaki** [comment]
Wait what.

File: src/components/navigation/NavigationMenu.vue [hunk 1]
```diff
@@ -80,65 +77,53 @@ import {
...
-                    this.$store.commit("closeGameRunningModal");
-                    this.$emit("error", error);
-                } else {
-                    throw error;
-                }
-            }
+</script>
 
-        }
+<style lang="scss">
```
Comment (ebkr) [inline src/components/navigation/NavigationMenu.vue:123]: Not 100% sure here but you may be able to change this to `<style lang="scss" scoped>` to keep the CSS strictly related to this component only

File: src/components/ExpandableCard.vue [hunk 2]
```diff
@@ -64,15 +64,6 @@
         @Prop({default: false})
         showSort: boolean | undefined;
 
-        @Prop({default: false})
```
Comment (ebkr) [inline src/components/ExpandableCard.vue:67]: Good spot 👍

**Review by ebkr — CHANGES_REQUESTED** [review]
One thing to resolve and another to verify (but not essential at all).

Otherwise this looks and works great, thanks 👍

File: src/components/navigation/NavigationMenu.vue (hunk 1 above)
Comment (anttimaki) [reply src/components/navigation/NavigationMenu.vue]: Nice, I wasn't aware that such a thing existed. Although I had to also add `::v-deep` [here](https://github.com/ebkr/r2modmanPlus/pull/889/files#diff-534c3ccf44736098d36e3858bdfb2bf282b0ece6e9f094aa8602c700a13cd64bR130) as per the [documentation](https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors) for the selector to work after that.

File: src/components/navigation/NavigationMenu.vue [hunk 3]
```diff
@@ -14,44 +14,44 @@
...
-                           class="tagged-link" :class="[view === 'installed' ? 'is-active' : '']">
-                            <i class="fas fa-folder tagged-link__icon icon--margin-right" data-ref="installed" @click.prevent.stop="emitClick($event.target)"/>
-                            <span class="tagged-link__content margin-right margin-right--half-width" data-ref="installed" @click.prevent.stop="emitClick($event.target)">Installed</span>
-                            <span class="tag tagged-link__tag" :class="[{'is-link': view !== 'installed'}]"
-                            data-ref="installed" @click.prevent.stop="emitClick($event.target)">{{localModList.length}}</span>
-                        </a>
+                        <router-link :to="{name: 'manager.installed'}" class="tagged-link">
+                            <i class="fas fa-folder tagged-link__icon icon--margin-right" />
+                            <span class="tagged-link__content">Installed</span>
+                            <span class="tag tagged-link__tag">{{localModList.length}}</span>
```
[the remark this answers is not in the document]
Comment (anttimaki) [reply src/components/navigation/NavigationMenu.vue]: Fixed this by [adding a method](https://github.com/ebkr/r2modmanPlus/pull/889/files#diff-534c3ccf44736098d36e3858bdfb2bf282b0ece6e9f094aa8602c700a13cd64bR93-R96) and calling it for [installed](https://github.com/ebkr/r2modmanPlus/pull/889/files#diff-534c3ccf44736098d36e3858bdfb2bf282b0ece6e9f094aa8602c700a13cd64bR20) and [online](https://github.com/ebkr/r2modmanPlus/pull/889/files#diff-534c3ccf44736098d36e3858bdfb2bf282b0ece6e9f094aa8602c700a13cd64bR32) mod counts.

**Pull request merged**